docker-compose.utils.hasura.yaml

```
@@ -1,61 +1,61 @@
-    depends_on:
-      hasura:
-        condition: service_healthy
-      postgres:
-        condition: service_healthy
-    volumes:
-      - ./hasura/init.sh:/init.sh:ro
-    entrypoint:
-      - /bin/sh
-      - -c
-      - "echo 'Waiting 60s for system to stabilize...' && sleep 60 && sh /init.sh"
+services:
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)